.github/workflows/main.yaml

```diff
@@ -1,7 +1,5 @@
 on:
   push:
-    branches:
```

Comment:
What's the reasoning here?

Reply:
oh, this was my debugging GH actions not working